libs/authorizenet/src/facades/authorize-net-facade.interface.ts

```diff
@@ -0,0 +1,12 @@
+import { Action } from '@ngrx/store';
+import { Observable } from 'rxjs';
+
+import { DaffAuthorizeNetTokenResponse } from '../models/response/authorize-net-token-response';
+
+export interface DaffAuthorizeNetFacadeInterface<T extends DaffAuthorizeNetTokenResponse = DaffAuthorizeNetTokenResponse> {
```

Comment:
This should extend from `DaffStoreFacade<Action>`

Reply:
done